app/views/admin/accounts/_account.html.haml

```diff
@@ -4,6 +4,9 @@
   %td.domain
     - unless account.local?
       = link_to account.domain, admin_accounts_path(by_domain: account.domain)
+  %td.protocol
+    - unless account.local?
+      = protocol_for_display(account.protocol)
```

Comment:
How about "OStatus"?  It's written as ostatus in code, but we want it to be transformed to OStatus. Replacing all occurence of `ostatus` to `o_status` is nonsense and impossible.